Lib/unittest/async_case.py

```diff
@@ -115,6 +115,7 @@ def _callMaybeAsync(self, func, /, *args, **kwargs):
     def _setupAsyncioRunner(self):
         assert self._asyncioRunner is None, 'asyncio runner is already initialized'
         runner = asyncio.Runner(debug=True)
+        asyncio.set_event_loop(runner.get_loop())
         self._asyncioRunner = runner
 
     def _tearDownAsyncioRunner(self):
```

Comment:
This still needs to call set_event_loop(None)

Reply:
Fixed, now it only sets the event loop if the policy registry was initialized which is done if no event loop factory was supplied.